src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java

```diff
@@ -51,8 +61,15 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
             }
         }
 
-        TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
-        player.setOp(true);
+         if (!player.isOp())
+         {
+         	TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
+         	player.setOp(true);
+         }
+         else
+         {
+         	playerMsg("You don't have permission to OP users that are already Opped.", ChatColor.YELLOW);

```

Comment:
Message here doesnt make sense or is even needed. Just display no message at all...